boards/silabs/radio_boards/slwrb4180b/slwrb4180b_defconfig

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_ARM_MPU=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_GPIO=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=38400000
+CONFIG_CMU_HFCLK_HFXO=y
+CONFIG_PINCTRL=y
```

Comment:
pinctrl not allowed in board files

Reply:
> pinctrl not allowed in board files

We are planning to fix all Silabs boards in one go, since this isn't just a board configuration thing, rather requires updates to existing drivers. IMO it'd be cleaner to fix this therefore separately when the drivers themselves are also fixed (which in turn doesn't have anything really to do with the addition of a new board definition, which this PR is about)

Reply:
This line can now be removed since #81764 was already merged